packages/cardano-graphql/src/WalletProvider/queryTransactions/graphqlTransactionsToCore.ts

```diff
@@ -32,7 +32,7 @@ const scriptToCore = (script: GraphqlScript): Cardano.Script => {
     return { native: nativeScriptToCore(script) };
   } else if (script.__typename === 'PlutusScript') {
     return {
-      plutus: script.cborHex
+      [script.type === 'PlutusScriptV1' ? ('plutus:v1' as keyof Cardano.Script) : 'plutus:v2']: script.cborHex
```

Comment:
I'm wondering if `('plutus:v1' as const)` would do the trick here

Reply:
![image](https://user-images.githubusercontent.com/303881/152306514-4c9397b8-45c8-46d7-88e9-f87714aaff69.png)